src/rdkafka.h

```diff
@@ -7920,13 +7920,14 @@ const char *rd_kafka_MemberDescription_client_id(
     const rd_kafka_MemberDescription_t *member);
 
 /**
- * @brief Gets group instance id of a \p member.
+ * @brief Gets group instance id of \p member.
  *
  * @param member The group member.
+ *
  * @return The group instance id, or NULL if not available.
```

Comment:
I'm using `@return` because it's the documented Doxygen command, [returns](https://doxygen.nl/manual/commands.html#cmdreturns) is an alias for that. Another reason is that when the editor auto generates the documentation structure, it uses `@return`  (though it can be changed)